docs/pages/versions/unversioned/sdk/url.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: URL
+description: Standard URL API available on all Expo-supported platforms. 
+sourceCodeUrl: 'https://github.com/expo/expo/tree/main/packages/expo'
+packageName: 'expo'
+---
+
+import PlatformsSection from '~/components/plugins/PlatformsSection';
+
+The standard URL API is available on all Expo-supported platforms.
+
+<PlatformsSection android emulator ios simulator web />
+
+## Installation
+
+This API is part of the `expo` package. On native platforms, the build-in support for `URL` and `URLSearchParams` replaces the shim in `react-native`. Refer to the [browser and server runtime support](https://caniuse.com/url) for web and Node.js.
+
+## Usage
+
+```js
+const url = new URL('https://expo.dev');
+
+const params = new URLSearchParams();
+```
+
+## Conformance
+
+Unicode characters are not supported on native platforms.
```

Comment:
Add more clarification about what this means and in what context. Also all characters, practically speaking, are Unicode characters.